src/deltaE/deltaEHCT.js

```diff
@@ -0,0 +1,42 @@
+import hct from "../spaces/hct.js";
+import {viewingConditions} from "../spaces/hct.js";
+
+const rad2deg = 180 / Math.PI;
+const deg2rad = Math.PI / 180;
+const ucsCoeff = [1.00, 0.007, 0.0228];
+
+
+function convertUcsAb (coords) {
+	// Convert HCT chroma and hue (CAM16 JMh colorfulness and hue) using UCS logic for a and b.
```

Comment:
Nit: shouldn't the comment of what the function does be above the function?

Reply:
Probably 😅.